packages/shared/src/utils/routes/routes.ts

```diff
@@ -61,6 +62,7 @@ export const routes = {
     p2p_my_ads: '/cashier/p2p/my-ads',
     p2p_my_profile: '/cashier/p2p/my-profile',
     p2p_advertiser_page: '/cashier/p2p/advertiser',
+    p2p_v2_inner: '/cashier/p2p-v2/inner',
```

Comment:
nit: also for this, we can create our own routes.ts file and define all the routes there, similar to wallets 👀

Reply:
yeah sure.